Events/LiftoffEvent.cs

```diff
@@ -36,10 +52,20 @@ static LiftoffEvent()
 
         public SignalSource nearestDestination { get; private set; }
 
-        public LiftoffEvent(DateTime timestamp, decimal? longitude, decimal? latitude, bool playercontrolled, SignalSource nearestDestination) : base(timestamp, NAME)
+        public long systemAddress { get; private set; }
+
+        public long? bodyId { get; private set; }
+
+        public LiftoffEvent(DateTime timestamp, decimal? longitude, decimal? latitude, string system, long systemAddress, string body, long? bodyId, bool? onStation, bool? onPlanet, bool playercontrolled, SignalSource nearestDestination) : base(timestamp, NAME)
         {
             this.longitude = longitude;
             this.latitude = latitude;
+            this.systemname = system;
+            this.systemAddress = systemAddress;
+            this.bodyname = body;
+            this.bodyId = bodyId;
+            this.onstation = onStation;
```

Comment:
inconsistent capitalization

Reply:
It's consistent with our standard practice of using lowercase for user facing variables and camel case for internal variables, but if you feel strongly about it then I can change it.

Reply:
I was unaware of that convention.